cmd/registry.go

```diff
@@ -42,3 +50,35 @@ func AvailableCommands() []string {
 	sort.Strings(avail)
 	return avail
 }
+
+// Register a boulder config struct.
+func RegisterConfig(name string, f *ConfigValidator) {
+	registry.Lock()
+	defer registry.Unlock()
+
+	if registry.config == nil {
+		registry.config = make(map[string]*ConfigValidator)
+	}
+
+	if registry.config[name] != nil {
+		panic(fmt.Sprintf("config %q was registered twice", name))
+	}
+	registry.config[name] = f
+}
+
+func LookupConfig(name string) *ConfigValidator {
+	registry.Lock()
+	defer registry.Unlock()
+	return registry.config[name]
+}
+
+func AvailableConfigs() []string {
```

Comment:
If you combine RegisterConfig and RegisterCommand into one function, then this helper goes away.

Reply:
I don't believe it does as it's used to populate the mapping on line `27` of `TestConfigValidation()`.